apps/sms/style/sms.css

```diff
@@ -1038,6 +1038,26 @@ html[dir="rtl"] .contact-header aside.pack-end {
   float: left;
 }
 
+/* ellipsis direction hack before bug 883884 fixed */
+.ellipsis-dir-fix {
+  display: inline-block;
+  max-width: 100%;
+
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+/* margin and line-height adjustment for inline-block bdi element */
```

Comment:
Please see https://github.com/steveck-chung/gaia/commit/79e556909298e2ae9c602187343fd92e1707211a#commitcomment-10044093 for the reason why I add this line-height and margin